force_torque_sensor_broadcaster/test/test_force_torque_sensor_broadcaster.cpp

```diff
@@ -0,0 +1,311 @@
+// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/*
+ * Authors: Subhas Das, Denis Stogl
+ */
+
+#include <stddef.h>
```

Comment:
I'd opt for removing this one but even if we *really* need to keep it, `cstddef` would be the name

https://en.wikibooks.org/wiki/C_Programming/stddef.h

Reply:
Removed.